.github/workflows/tsc.yml

```diff
@@ -7,6 +7,8 @@ on:
       - '**.ts'
       - '**.tsx'
       - '**/tsc.yml'
+      - 'package.json'
+      - 'yarn.lock'
```

Comment:
Maybe `**.lock`?

Reply:
That will also trigger Gemfile.lock and Podfile.lock, is that something we want? :)